api/v1alpha1/broom_types.go

```diff
@@ -41,33 +41,51 @@ const (
 )
 
 type BroomAdjustment struct {
-	Type  BroomAdjustmentType `json:"type"`
-	Value string              `json:"value"`
+	Type     BroomAdjustmentType `json:"type"`
+	Value    string              `json:"value"`
```

Comment:
Could you write godoc like README?

```suggestion
	// Adjustment value. For `Add`, it is the value to add to the current limit 
	Value    string              `json:"value"`
```